tests/wicked/advanced/sut/t17_aggregate_both_cards_legacy.pm

```diff
@@ -29,9 +29,7 @@ sub run {
     $self->wicked_command('ifup', $sut_iface);
     validate_script_output('ip a s dev ' . $ctx->iface(),  sub { /SLAVE/ });
     validate_script_output('ip a s dev ' . $ctx->iface2(), sub { /SLAVE/ });
-    my $remote_ip = $self->get_remote_ip(type => 'host');
-    $self->ping_with_timeout(ip => $remote_ip, interface => $sut_iface);
-    validate_script_output("ping -c30 $remote_ip -I $sut_iface", sub { /0% packet loss/ });
+    $self->ping_with_timeout(type => 'host', interface => $sut_iface, count_success => 5, timeout => 4);
```

Comment:
This doesn't require a change in this PR:
* I think the `count_success` argument is now misleading. Cause we only fail, if more then 50% of `count_success-1` failed.
Maybe a topic for https://progress.opensuse.org/issues/52541 , WDYT?